analyzer/src/main/kotlin/managers/Yarn.kt

```diff
@@ -33,7 +35,15 @@ import org.ossreviewtoolkit.analyzer.managers.utils.mapDefinitionFilesForYarn
 import org.ossreviewtoolkit.model.config.AnalyzerConfiguration
 import org.ossreviewtoolkit.model.config.RepositoryConfiguration
 import org.ossreviewtoolkit.model.jsonMapper
+import org.ossreviewtoolkit.utils.common.DiskCache
 import org.ossreviewtoolkit.utils.common.Os
+import org.ossreviewtoolkit.utils.ort.ortDataDirectory
+
+private val yarnInfoCache = DiskCache(
+    directory = ortDataDirectory.resolve("cache/yarn-info"),
```

Comment:
Should we use `cache/analyzer/yarn-info` instead?

Note: If I'd be debugging a random "analyzer reports wrong meta-data" bug report, I'd probably start with reproducing trying to delete the whole analyzer cache first. Also I'd probably ask the user if it reproduces with the cache deleted. That deletion step would become simpler then.

Reply:
> Should we use `cache/analyzer/yarn-info` instead?

I'd be fine with that, but I'd then like to move Maven's "cache/remote_artifacts" accordingly.

Reply:
Oh, wasn't aware of that one. Maybe then

`cache/analyzer/maven/remote-artifacts`
`cache/analyzer/yarn/yarn-info`

This allows (1) simple handling of all analyzer cache, e.g. delete or docker mount and (2) simple handling of all cache data for a specific package manager, e.g. debugging / deleting.

Reply:
Also fine with me, although I probably would say just "cache/analyzer/yarn/info" then to not repeat "yarn".

Reply:
I've made these changes, please re-review @oss-review-toolkit/kotlin-devs.